src/lib/appConfig.ts

```diff
@@ -1,31 +1,105 @@
-import { LocalStorage } from "quasar"
-import { AppConfig, Plot, emptyAppConfig, SegmentCache } from "./util"
+import { Dialog, DialogChainObject } from "quasar"
+import { appName } from "./util"
+import * as path from "@tauri-apps/api/path"
+import * as fs from "@tauri-apps/api/fs"
+
+export interface SegmentCache {
+  networkSegmentCount: number
+  blockchainSizeGB: number
+}
+export interface Plot {
+  location: string
+  sizeGB: number
+}
+
+interface Config {
+  [index: string]: any
+  plot: Plot
+  rewardAddress: string,
+  launchOnBoot: boolean,
+  version: string,
+
+  // TODO: remove this after PlottingProgress refactoring
+  segmentCache: SegmentCache,
+}
+
+const emptyConfig: Config = {
+  plot: { location: "", sizeGB: 0 },
+  rewardAddress: "",
+  launchOnBoot: true,
+  version: process.env.APP_VERSION || "?",
```

Comment:
don't we assume `APP_VERSION` is always a string? we never display `"?"`

Reply:
the reason is, `process.env.APP_VERSION` return type is `string | null` or `string | undefined`. 
If the `APP_VERSION` cannot be resolved, there must be something wrong. That's why I put the "?", and made `version` accept type of `string` only.

Reply:
can do `process.env.APP_VERSION as string`